@extends('layouts.errors')
@section('content')
	<div id="content" class="site-content"><div class="container"><div class="inner-wrapper">    
	<div id="primary" class="content-area">
		<main id="main" class="site-main" role="main">
            <article class="post type-post status-publish format-standard has-post-thumbnail hentry category-events">
                <header class="entry-header">
		            <h2 class="entry-title">Сайт отключен!</h2></header>
                        <div class="entry-content">
							<h3>Уважаемые пользователи!<br />
							<p>В связи с проведением технологических работ возможно кратковременное ограничение доступа к сайту.</p>
							</h3> 
                        </div>
	            <footer class="entry-footer">Пожалуста, попробуйте зайти на сайт позже.</footer>
            </article>
		</main>
	</div>
  </div>
 </div>
</div>     
@endsection